src/main/java/step1/domain/StringAddCalculator.java

```diff
@@ -6,22 +6,19 @@
 import java.util.regex.Pattern;
 
 public class StringAddCalculator {
-    private static final int ZERO = 0;
     private static final String DEFAULT_SEPARATOR = ",|:";
     private static final String CUSTOM_SEPARATOR = "//(.)\n(.*)";
-
+    private static Pattern pattern;
```

Comment:
인스턴스 변수로만 선언하고 객체 생성을 안하고 있네요. 이런 경우라면 여전히 객체생성은 splite() 메서드가 호출 될떄 마다 실행됩니다.

```suggestion
    private static final Pattern PATTERN = Pattern.compile(CUSTOM_SEPARATOR);
```
이처럼 패턴을 미리 '생성'하는 것에 초점을 맞춰주세요. 🙂